src/components/battle/select/index.tsx

```diff
@@ -29,21 +33,24 @@ function Select() {
               <>
                 <BattleMusicInfo
                   music={musicData?.challenged.music}
-                  onClick={(e) => onClickMusic(e, 'left')}
+                  onClick={(e) => onClickMusic(e, 'left', musicData.battleId, musicData?.challenged.postId)}
                   clickSide='left'
                 />
                 <BattleMusicInfo
                   music={musicData?.challenging.music}
-                  onClick={(e) => onClickMusic(e, 'right')}
+                  onClick={(e) => onClickMusic(e, 'right', musicData.battleId, musicData?.challenging.postId)}
                   clickSide='right'
                 />
               </>
             )}
           </BattleContainer>
         </Section>
-        <Skip onClick={onClickSkip}>건너뛰기</Skip>
+        {!battleId && <Skip onClick={onClickSkip}>건너뛰기</Skip>}
```

Comment:
```suggestion
    {battleId ?? <Skip onClick={onClickSkip}>건너뛰기</Skip>
```
이거랑 같은 뜻이라면 이렇게 쓰는 것도 좋을 것 같아요~